Resizin/Classes/ResizinSettings.swift

```diff
@@ -114,6 +122,11 @@ public struct ResizinSettings {
         }
     }
     
+    /// Output format
+    ///
+    /// `nil` value returns image in original format
+    public var outputFormat: OutputFormat?
```

Comment:
❓ Just space for discussion - do we want output format to be optional? Maybe it would make sense to make it mandatory? What do you and others think?

Reply:
Yes, it makes sense, so I added a new case `original,` which does not have `stringValue`.